config/routes.rb

```diff
@@ -18,21 +18,20 @@
     get "/test-search/search/opensearch" => "search#opensearch"
   end
 
-  # Routes for the for Transition/Brexit Checker
```

Comment:
Did you mean to get rid of this comment?
I think it's there because this is all wierd tech debt (an app nested in an app), might be helpful to leave it highlighed?

Reply:
Yes, it was deliberate.  I think now the routes are in a scope, their purpose are probably self-explanatory from the route `/transition-check`.

Reply:
Fair enough, I guess it's less likely we'll rebrand the entire enterprise again this year.